detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/parsing/MavenProjectInspectorDetectable.java

```diff
@@ -27,6 +29,7 @@ public class MavenProjectInspectorDetectable extends Detectable {
     private final ProjectInspectorResolver projectInspectorResolver;
     private final ProjectInspectorExtractor projectInspectorExtractor;
     private final ProjectInspectorOptions projectInspectorOptions; // TODO: Options don't belong here
+    public String includeShadedDependencies = "";
```

Comment:
From what I've seen we prefer to keep variables private and expose setters for them.